src/api/users.ts

```diff
@@ -1,5 +1,76 @@
 import { Router } from "express";
-
 const router = Router();
+const express = require('express');
+const jwt = require('jsonwebtoken')
+const bcrypt = require('bcryptjs');
```

Comment:
For this project, we are using the ESM `import`/`export` syntax. Also, the latest changes includes methods for this purpose.